holopy/inference/tests/common.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2011-2016, Vinothan N. Manoharan, Thomas G. Dimiduk,
+# Rebecca W. Perry, Jerome Fung, and Ryan McGorty, Anna Wang
+#
+# This file is part of HoloPy.
+#
+# HoloPy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# HoloPy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with HoloPy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+
+from holopy.inference import prior
+from holopy.inference.model import Model
+from holopy.scattering import Sphere
+
+class SimpleModel(Model):
```

Comment:
Why is SimpleModel defined both here and in `test_cma`? Am I missing something?

Reply:
I moved it but forgot to remove the original. Fixed.